chapter-70-missiles/src/inventory_system/remove_system.rs

```diff
@@ -31,4 +31,4 @@ impl<'a> System<'a> for ItemRemoveSystem {
 
         wants_remove.clear();
     }
-}
```

Comment:
Looks like I didn't fix all the missing newline edits -- I'll revisit this tomorrow